README.md

```diff
@@ -55,3 +55,18 @@ or change your directory to the infrastructure project, and then run the command
 cd .\src\Digdir.Domain.Dialogporten.Infrastructure\
 dotnet ef migrations add TestMigration
 ```
+
+## Development in local and test environments
+To generate test tokens see https://github.com/Altinn/AltinnTestTools. There is a request in the postmann collection for this.
```

Comment:
```suggestion
To generate test tokens see https://github.com/Altinn/AltinnTestTools. There is a request in the Postman collection for this.
```

Ser det er en "throguh" ("through") lenger opp her og .. men det er ikke mulig å legge til suggestions for deler av fila du ikke har vært borti?

Reply:
Hehe! Jeg fikser det. Sagt det før, kommer til å si det igjen - det er helt nydelig at du ser og retter alle skrivefeilene mine! 🎉